lib/discourse_translator/post_extension.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module DiscourseTranslator
+  module PostExtension
+    extend ActiveSupport::Concern
+
+    prepended { before_update :clear_translator_custom_fields, if: :raw_changed? }
+
+    private
+
+    def clear_translator_custom_fields
+      return if !SiteSetting.translator_enabled
+
```

Comment:
I asked in another PR but lost track of it 😄 - can we skip these checks in class extensions?